### PR TITLE
VB-596 Refactored migration scripts, and changed table/entities from …

### PR DESCRIPTION
**Pull request by AledWEvans**
…support to support type, also changed name from AvailableSupport.kt to SupportTypeDto.kt and SupportDto.kt to VisitorSupportDto.kt.

## What does this pull request do?

This code change tidied up migrations scripts, and refactored some names of DB tables and associated entities 

## What is the intent behind these changes?

to improve maintainability and readability of code

### CODE REVIEW

**Comment by rj-adams** [comment]
@AledWEvans - can we discuss.

Many of these files should not be in this repo - they should be removed.

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/data/SupportTypeDto.kt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package uk.gov.justice.digital.hmpps.visitscheduler.data
 
 import io.swagger.v3.oas.annotations.media.Schema
-import uk.gov.justice.digital.hmpps.visitscheduler.jpa.Support
+import uk.gov.justice.digital.hmpps.visitscheduler.jpa.SupportType
 
-@Schema(description = "Available Support")
-data class AvailableSupport(
+@Schema(description = "Support Type")
+data class SupportTypeDto(
```
Comment (philip-milne) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/data/SupportTypeDto.kt:7]: Good clarification

**Review by philip-milne — COMMENTED** [review]
Great stuff, LGTM.  Possibly update the api schema to match the improvements.

**Review by philip-milne — APPROVED** [review]
LGTM

**Pull request merged**